src/project/.lisa/skills/github/SKILL.md

```
@@ -398,18 +399,110 @@ lisa github sync --repo owner/repo
+
+1. **Identify the failure** - Check CircleCI logs or GitHub checks
+2. **Push a fix** - Commit and push the fix to the branch
+3. **Retrigger tests** - Toggle the TEST label:
+   ```bash
+   gh pr edit <PR_NUMBER> --repo <owner/repo> --remove-label "TEST" && \
+   sleep 2 && \
+   gh pr edit <PR_NUMBER> --repo <owner/repo> --add-label "TEST"
+   ```
+4. **Monitor** - Watch for the new pipeline to complete
+
 ## Cross-model checklist
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)